<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NScoreDialog</class>
 <widget class="QDialog" name="NScoreDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Normal score transforms variables so that its distribution becomes normal, that is, a gaussian distribution with mean = 0 and standard deviation = 1. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblVariable">
     <property name="text">
      <string>Variables:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>1) Configure and run:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnParams">
       <property name="toolTip">
        <string>review nscore parameters</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/setting16</normaloff>:/icons/setting16</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>2) Evaluate results:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHistogram">
       <property name="toolTip">
        <string>resulting histogram</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/histo16</normaloff>:/icons/histo16</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;3) Save normal variable and transform table:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="toolTip">
        <string>save n-score variable and transform table</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/save16</normaloff>:/icons/save16</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDismiss">
       <property name="text">
        <string>Dismiss</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnDismiss</sender>
   <signal>clicked()</signal>
   <receiver>NScoreDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnParams</sender>
   <signal>clicked()</signal>
   <receiver>NScoreDialog</receiver>
   <slot>onParams()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnHistogram</sender>
   <signal>clicked()</signal>
   <receiver>NScoreDialog</receiver>
   <slot>onHistogram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>NScoreDialog</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onParams()</slot>
  <slot>onHistogram()</slot>
  <slot>onSave()</slot>
 </slots>
</ui>
